src/screens/Elections/index.js

```diff
@@ -22,4 +31,214 @@ const propTypes = {
   }),
 };
 
+const distinctPositions = new Set()
```

Comment:
I don't think you should have this variable outside the function.  It will not reset on each render, which I think will cause it to keep adding each time you load the page.  It would be better to do that mapping in the mapStateToProps of the container. You could end up with a list of elections, and the candidates in each already mapped, to vastly simplify your view component.  ex: 
```
[ { electionId: 'governor', candidates: [ {...candidate}, {...candidate} },
{ electionId: 'district4 senator', candidates: [ {...candidate}, {...candidate} },
...
]
```
or something